lib/init/variables.html

```diff
@@ -331,12 +331,12 @@ <h2>List of selected (GRASS related) shell environment variables</h2>
   <dt>GRASS_RASTER_TMPDIR</dt>
   <dd>[rasterlib]<br> By default the GRASS temporary directory is
   located in <tt>$LOCATION/$MAPSET/.tmp/$HOSTNAME</tt>. If
-  GRASS_RASTER_TMPDIR is set to some existing directory, this directory 
-  will be used to store data files of newly created raster maps. This 
-  can be usefull if this temporary directory is located on a fast 
-  storage device. However, upon closing a new raster map, these files 
-  need to be transferred to the final destination (the current mapset) 
-  which might be slow or even slower than creating the temorary raster 
+  GRASS_RASTER_TMPDIR is set to some existing directory, this directory
+  will be used to store data files of newly created raster maps. This
+  can be usefull if this temporary directory is located on a fast
+  storage device. However, upon closing a new raster map, these files
+  need to be transferred to the final destination (the current mapset)
+  which might be slow or even slower than creating the temorary raster
   data files directly on the final storage device.</dd>
```

Comment:
This is indeed a good point and important documentation.

What I am wondering is, if this new functionality could be used specifically for really temporary data (like from `r.terraflow` or `r.watershed`, or temporary text files, ...)?

Will this f.ex. be available in g.tempfile?

Reply:
> What I am wondering is, if this new functionality could be used specifically for really temporary data (like from `r.terraflow` or `r.watershed`, or temporary text files, ...)?

Yes, this PR was designed for modules creating potentially large temporary data. Modules such as `r.terraflow`, `r.watershed`, `i.segment`, `g.tempfile` would need a new option where to create files for temporary data.

However, the most efficient way to speed up data processing with final data stored on a slow block device is to create a temporary mapset on a fast block device, if available, and copy the final results to the slow block device.
> 
> Will this f.ex. be available in g.tempfile?

If `g.tempfile` becomes a new option for the base directory of temporary files, yes.

Reply:
Nice!